tests/acceptance/00_basics/06_host_specific_data/10-variable-references.cf.json

```diff
@@ -0,0 +1,14 @@
+{
+    "variables" : {
+        "test_variable": {
+            "value" : "$(sys.uptime)",
+            "comment" : "comment1",
+            "tags" : ["test_tag"]
+        }
+        "test_variable2": {
```

Comment:
github clued me in to a missing comma between the variables with a red `:`, invalid JSON but parseable by our parser eh?

Reply:
Good catch! There was also one extra comma in the *02-variables.cf.json* file.